app/server/lib/DocApi.ts

```diff
@@ -573,6 +565,14 @@ export class DocWorkerApi {
       })
     );
 
+    this._app.get('/api/docs/:docId/attachments/stores', isOwner,
+      withDoc(async (activeDoc, req, res) => {
+        const configs = await getConfiguredAttachmentStoreConfigs();
+        const labels = configs.map(c => ({label: c.label}));
+        res.json(labels);
```

Comment:
For new endpoints, can you return an object? If it is a list, can the list be nested as a member? It makes it easier to add other new members later (e.g. for paging or whatever) without breaking compatibility.